aws/cloudformation/components/ami.yml.erb

```diff
@@ -177,7 +187,6 @@ frontend_device_name ||= '/dev/sda1'
 
           # Signal CompleteLifecycleAction, in case this instance was launched from an Auto Scaling process.
           LIFECYCLE_HOOK=WebServerHook-${AWS::StackName}
-          AUTO_SCALING_GROUP=Frontends-${AWS::StackName}
```

Comment:
moving this declaration up in the file, since its also used in the `aws ec2 crreate-tags` call